src/FurCoNZ.Web/Controllers/AccountController.cs

```diff
@@ -33,18 +36,30 @@ public async Task<IActionResult> Index(CancellationToken cancellationToken = def
             return View(new AccountViewModel(await _userService.GetCurrentUserAsync(cancellationToken)));
         }
 
-        [HttpPost]
+        [HttpGet, HttpPost]
```

Comment:
CSRF checks aren't performed on GET requests, can you confirm that this won't be affected by CSRF issues?